packages/webpack-common/src/extractText.js

```diff
@@ -18,7 +18,7 @@ export {
  */
 function getLoaderConfigByType (context, webpackConfig, fileType) {
   const loaderConfig = webpackConfig.module.loaders.find(
-    (loader) => loader.test === context.fileType(fileType)
+    (loader) => regexEqual(loader.test, context.fileType(fileType))
```

Comment:
Hmm. I propose we first investigate why this is happening. Looks like an easy fix to deep-compare the regexes, but that might be a minor bug in webpack-merge. If so we should not try to fix it afterwards in webpack-blocks, but fix it in webpack-merge.